dts/bindings/pinctrl/gd,gd32-pinctrl-af.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021 Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Gigadevice GD32 pin controller (AF model).
+
+    All pin configurations should be defined as children of the pinctrl node.
+    It is highly recommended to use pre-defined macros for the pinmux field. If
+    the SoC being used does not have pre-defined macros yet, the GD32_PINMUX_AF
+    helper macro can be used to manually define the pinmux entries.
+
+    The representation used by GD32 pinctrl driver is detailed in the example
+    below.
+
+    /** board-pinctrl.dtsi file **/
+
+    /* include pre-defined combinations for the SoC variant used by the board */
+    #include <dt-bindings/pinctrl/gd32f450i(g-i-k)xx-pinctrl.h>
+
+    &pinctrl {
+      /* configuration for the usart0 "default" state */
+      usart0_default: usart0_default {
+        /* group 1 */
+        pins1 {
+          /* configure PA9 as USART0 TX and PA11 as USART0 CTS */
+          pinmux = <USART0_TX_PA9>, <USART0_CTS_PA11>;
+        };
+        /* group 2 */
+        pins2 {
+          /* configure PA10 as USART0 RX and PA12 as USART0 RTS */
+          pinmux = <USART0_RX_PA10>, <USART0_RTS_PA12>;
+          /* both PA10 and PA12 have pull-up enabled */
+          bias-pull-up;
+        };
+      };
+
+    The list of supported standard properties:
+
+    - drive-push-pull: Push-pull drive mode (default, not required).
+    - drive-open-drain: Open-drain drive mode.
+    - bias-disable: Disable pull-up/down (default, not required).
+    - bias-pull-up: Enable pull-up resistor.
+    - bias-pull-down: Enable pull-down resistor.
+    - slew-rate: Set the maximum speed (and so the slew-rate) of the output
+      signal (default: 2MHz).
```

Comment:
bias-high-impedance is missing

Reply:
ditto

Reply:
> bias-disable: Disable pull-up/down (default, not required).

What is this? Analog or High-Impedance? Again, it is not clear the meaning and it is possible improve here to a very clear pin state.